_layouts/index.html

```diff
@@ -54,74 +54,88 @@
       <div id="informations">
         <ul>
           <li>
+            <h2>How Do I Install Homebrew?</h2>
```

Comment:
It's probably not the nicest thing, but you can check if the string is available and default to the english string: https://github.com/jaimeMF/homebrew.github.io/commit/49ec3470df2631e267c9637b2709f846ffbd2964. If someone browsing in another language notices it, he can submit the translation.